src/Compilers/CSharp/Portable/FlowAnalysis/AbstractFlowPass.cs

```diff
@@ -748,39 +729,20 @@ private void ResolveBreaks(TLocalState breakState, LabelSymbol label)
         /// </summary>
         private void ResolveContinues(LabelSymbol continueLabel)
         {
-            var pendingBranches = PendingBranches;
-            var count = pendingBranches.Count;
-
-            if (count != 0)
+            var pendingBranches = PendingBranches.GetAndRemoveBranches(continueLabel);
```

Comment:
>var pendingBranches = PendingBranches.GetAndRemoveBranches(continueLabel); [](http://example.com/codeflow?start=12&length=74)

It looks like this function is doing exactly what the beginning of the previous function is doing. Consider if extracting a shared helper would make sense.